docs/tutorials/tacho-motors.md

```diff
@@ -151,6 +153,8 @@ change.
     $ echo run-to-abs-pos > $MC/command
     $ while true; do echo -en "\033[0G$(cat $MC/position)   "; done
 
+Note: For `run-to-abs-pos` command only the absolute value of `duty_cycle_sp` or `speed_sp` matters. The direction of movement is determined automatically.
```

Comment:
:+1: we should add this to the source code documentation as well. 

* https://github.com/ev3dev/lego-linux-drivers/blob/master/motors/tacho_motor_class.c#L75
* https://github.com/ev3dev/lego-linux-drivers/blob/master/motors/tacho_motor_class.c#L123